### ISSUE TITLE
index.html

### ISSUE THREAD

**Issue by spremo123** [spremo123/flexbox#1]
<!DOCTYPE html><html><head><meta charset="utf-8"/><title>flexbox</title><meta content="width=device-width, initial-scale=1" name="viewport"/><link href="css/style.css" rel="stylesheet" type="text/css"/><script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js" type="text/javascript"></script><script type="text/javascript">WebFont.load({  google: {    families: ["Montserrat:100,100italic,200,200italic,300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic"]  }});</script><script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script><link href="https://uploads-ssl.webflow.com/img/favicon.ico" rel="shortcut icon" type="image/x-icon"/><link href="https://uploads-ssl.webflow.com/img/webclip.png" rel="apple-touch-icon"/></head><body><div class="section"><div class="col-1"><div data-animation="slide" data-duration="500" data-infinite="1" class="slider w-slider"><div class="w-slider-mask"><div class="slide w-slide"></div><div class="slide-2 w-slide"></div><div class="slide-3 w-slide"></div><div class="slide-4 w-slide"></div></div><div class="left-arrow w-slider-arrow-left"><div class="arrow-icon-2 w-icon-slider-left"></div></div><div class="right-arrow w-slider-arrow-right"><div class="arrow-icon w-icon-slider-right"></div></div><div class="slide-nav w-slider-nav w-round"></div></div><div class="slide-buttons-bg"></div></div><div class="col-2"><div class="text-wrapper"><h1 class="h1-heading">Caribbean Green<br/><span class="text-span">&amp; Blue Charcoal</span></h1><p class="paragraph">Welcome to the Color, Layout and Type Series. For this layout I decided to use the colors Caribbean Green and Blue Charcoal. The typeface is Futura from Typekit. The images are from Unsplash and have been converted to duotone colors via ShapeFactory.</p></div><a href="#" class="logo-text">Webdev For You<br/>Color, Layout, And Type Series</a><div class="menu-trigger-wrapper"><div class="menu-line"></div><div class="menu-line second"></div><div class="menu-line third"></div></div><a href="#" class="cta-wrapper w-inline-block"><div class="cta-text">View Color Combination @ ShapeFactory</div><img src="https://uploads-ssl.webflow.com/5d8f3f46b2bf2b97d8949a5f/5d8f695ed12b5b07748b34ab_5c4fba3abca6e1295df551e1_arrow-blue.png" alt=""/></a></div><div class="col-3"><div class="col-3-duotone"><h3 class="h3-heading">duotone<br/>‍</h3><a href="#" class="cta-wrapper duotone w-inline-block"><div class="cta-text">View Duotone Images</div><img src="https://uploads-ssl.webflow.com/5d8f3f46b2bf2b97d8949a5f/5d8f695ed12b5b07748b34ab_5c4fba3abca6e1295df551e1_arrow-blue.png" alt="" class="image"/></a></div><div class="col-3-gradient"><h3 class="h3-heading gradient">gradient<br/>‍</h3><a href="#" class="cta-wrapper gradient w-inline-block"><div class="cta-text gradient">View Gradient</div><img src="https://uploads-ssl.webflow.com/5d8f3f46b2bf2b97d8949a5f/5d8f70dae6be58d9833a701f_5c4fbc8235dd14129ff94f57_arrow-green.png" alt="" class="image"/></a></div></div></div><div class="menu-wrapper"><div class="menu-text-wrapper"><h1 class="heading">Layout</h1><ul class="list"><li class="list-item"><span class="text-span-2">Color 1:</span> Caribbean Green, HEX #00F1B3<br/></li><li class="list-item"><span class="text-span-3">Color 2:</span> Blue Charcoal, HEX #000916<br/></li><li class="list-item"><span class="text-span-4">Layout:</span> Flexbox<br/></li><li class="list-item"><span class="text-span-5">Type:</span> Futura<br/></li><li class="list-item"><span class="text-span-6">Images:</span> Unsplash.com<br/></li><li class="list-item"><span class="text-span-7">Duotone:</span> Shapefactory.co<br/></li></ul><div class="closebtn"><div class="menu-line"></div></div></div></div><script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.4.1.min.220afd743d.js" type="text/javascript" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><script src="https://uploads-ssl.webflow.com/5d8f3f46b2bf2b97d8949a5f/js/webflow.085ac1e77.js" type="text/javascript"></script></body></html>
